libsolutil/CDCL.h

```diff
@@ -68,25 +132,22 @@ class CDCL
 	std::pair<Clause, size_t> analyze(Clause _conflictClause);
 	size_t currentDecisionLevel() const { return m_decisionPoints.size(); }
 
-	void addClause(Clause _clause);
+	void addClause(const Clause& _lits);
```

Comment:
```suggestion
	void addClause(Clause const& _lits);
```
the type always comes first

Reply:
Oops, good find, sorry!